awx/ui_next/src/api/models/Users.js

```diff
@@ -24,6 +24,16 @@ class Users extends Base {
       params,
     });
   }
+
+  readRoles(userId, params) {
+    return this.http.get(`${this.baseUrl}${userId}/roles/`, {
+      params
+    });
+  }
+
+  roleOptions(userId) {
```

Comment:
nitpick: for options requests directly on the model's endpoint, we use a method called [readOptions](https://github.com/ansible/awx/blob/devel/awx/ui_next/src/api/Base.js) so it would be better to name this method `readRoleOptions`.